index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
```

Comment:
Node 4 supports `let`. We're not trying to fix any other ES6 bit beyond `import`/`export`. So if they use Node 4 they get the ES6 support of Node 4. If they use Node 7 they get the added ES6/7 support of Node 7.